src/pallene/gc.lua

```diff
@@ -28,77 +29,234 @@ local types = require "pallene.types"
 --   2) Identify functions that don't call the GC (directly or indirectly) and don't treat calls to
 --      them as potential GC sites. (Function inlining might mitigate this for small functions)
 --
---   3) Use a flow-based liveliness analysis to precisely identify the commands that a variable
---      appears live at, instead of approximating with first definition and last use.
---
---   4) Use SSA form or some form of reaching definitions analysis so that we we only need to mirror
+--   3) Use SSA form or some form of reaching definitions analysis so that we we only need to mirror
```

Comment:
Perhaps investigating the items on this list could be an interesting experiment for the project... We could do a subsequent PR for that and then compare how the benchmarks fare, before and after.

Reply:
Yes! We could start with them.